src/WCSimRunAction.cc

```diff
@@ -83,7 +83,10 @@ void WCSimRunAction::BeginOfRunAction(const G4Run* /*aRun*/)
     fSettingsOutputTree->Branch("WCDetCentre", WCDetCentre, "WCDetCentre[3]/F");
     fSettingsOutputTree->Branch("WCDetRadius", &WCDetRadius, "WCDetRadius/F");
     fSettingsOutputTree->Branch("WCDetHeight", &WCDetHeight, "WCDetHeight/F");
- 
+#ifdef GIT_HASH
+    const char* gitHash = GIT_HASH;
+    fSettingsOutputTree->Branch("GitHash", (void*)gitHash, "GetHash/C");
```

Comment:
There's a typo here... `GetHash/C` should be `GitHash/C`
I guess we should fix on nuPRISM/WCSim first so this PR gets updated